Mathlib/SetTheory/Cardinal/Basic.lean

```diff
@@ -833,6 +833,17 @@ theorem mk_subtype_mono {p q : α → Prop} (h : ∀ x, p x → q x) :
     #{ x // p x } ≤ #{ x // q x } :=
   ⟨embeddingOfSubset _ _ h⟩
 
+lemma card_ssubset' {A B : Set α} (hfin : #B < ℵ₀) (hlt : A ⊂ B) : #A < #B := by
+  have : #A ≤ #B := mk_subtype_mono hlt.1
+  have : Fintype A := ((lt_aleph0_iff_fintype).1 <| this.trans_lt hfin).some
+  have : Fintype B := ((lt_aleph0_iff_fintype).1 hfin).some
+  simpa using Finset.card_lt_card <| Set.toFinset_ssubset_toFinset.mpr hlt
+
+lemma card_ssubset {A B : Set α} (hfin : #A < ℵ₀) (hlt : A ⊂ B) : #A < #B := by
```

Comment:
This should named something like `card_lt_card`, analogous to the cardinal and finset versions.